src/Screens/History/History.js

```diff
@@ -0,0 +1,84 @@
+import "./history.css";
+import Movie from "./../../Components/Movie/Movie";
+import { useProducts } from "./../../Context/products-context";
+import { useEffect } from "react";
+import { useToken } from "./../../Context/token-context";
```

Comment:
try barrel exports

Reply:
can also set up config files to have absolute imports